core/src/main/java/org/elasticsearch/gateway/TransportNodesListGatewayStartedShards.java

```diff
@@ -191,6 +183,7 @@ public ShardId shardId() {
         public void readFrom(StreamInput in) throws IOException {
             super.readFrom(in);
             shardId = ShardId.readShardId(in);
+            List<DiscoveryNode> discoveryNodes = in.readList(DiscoveryNode::new);
```

Comment:
oops. left over from the first version I had. This request is never serialized - it is sent on the master which executes it locally, if someone would have done it from a transport client, it would be different.